model/src/main/scala/grasshopper/model/Status.scala

```diff
@@ -0,0 +1,14 @@
+package grasshopper.model
+
+import java.time.Instant
+
+trait ServiceStatus
+
+case class Status(status: String, service: String, time: String, host: String) extends ServiceStatus
```

Comment:
Should we even allow `host` to be passed in?  Seems like something we'd want to set once when the class is instantiated, and just reuse.